client/webpack.config.js

```diff
@@ -11,18 +11,19 @@ const config = {
     'babel-polyfill',
     './app/bundles/ConifyClient/startup/App',
   ],
-
+  resolve: {
+    //Allow files to be referenced relative ot the root directory of the bundle
+    fallback: [
+      path.join(__dirname,'app/bundles/ConifyClient'),
+    ],
+  },
   output: {
     filename: 'webpack-bundle.js',
     path: '../app/assets/webpack',
   },
 
   resolve: {
     extensions: ['', '.js', '.jsx'],
-    alias: {
-      react: path.resolve('./node_modules/react'),
-      'react-dom': path.resolve('./node_modules/react-dom'),
-    },
```

Comment:
node_modules is searched anyway for these modules, they don't need to be explicit.